README.md

```diff
@@ -35,66 +39,85 @@ project's tests. (Note that `build` depends on `check` which depends on the `jac
 
 #### Configuration
 
-Code coverage requirements can be specified for different scopes:
-- average coverage in a report
-- average coverage in a package
-- coverage in a class
-- coverage in a file
+Code coverage requirements are specified separately for the different "realms" reported by Jacoco:
+- average coverage in a report, specified by the `report` keyword
+- average coverage in a package, specified by the `_package` keyword
+- coverage in a class, specified by the `_class` keyword
+- coverage in a file, specified by the `file` keyword
 
-Scopes are specified either by their exact name as it appears in the Jacoco XML report, for example package scope
-`"org/product/module"`, file scope `"MyClass.java"`, class scope `"org/product/module/MyCLass"`), or by a regular
-expression matching those scope names, for example `~"org.*"` for all packages or class scopes starting with `org`.
+Note that the keywords for package and class realms require a `_` prefix since they are reserved words in Groovy. Each 
+realm contains a number of so-called "scopes": 
+- The file realm contains one scope for every source file, named by file name, e.g., "MyClass.java"
+- The class realm contains one scope for every (top-level, inner, or anonymous) class, named by fully qualified class
+name, e.g., "org/package/MyClass$1"
+- The package realm contains one scope for every package, named by the package name, e.g., "org/package"
```

Comment:
is it `/` delimited or `.` delimited? my guess is that packages are `.` delimited